sscpac/chat-locker
# Fixes #73 Removes JedisSettings and replaces with RocketChat.settings.Adds admin username setting that sets specified username as admin

To verify:
1. Login as testadmin.  You should see the "Administration" menu option when you expand the left navigation (top left next to testadmin username).
2. Click on Administration option
3. We added "MAC: menu option to set network classification and system country code.  
4. You can check mongo that only the 'testadmin' user has admin:true property.
